main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1597,21 +1599,45 @@ private static void getExtraOnlineInfo(@NonNull final Geocache cache, final Stri
         final String userToken = parseUserToken(page);
         final Observable<LogEntry> logs = getLogs(userToken, Logs.ALL);
         final Observable<LogEntry> ownLogs = getLogs(userToken, Logs.OWN).cache();
-        final Observable<LogEntry> specialLogs = Settings.isFriendLogsWanted() ?
-                Observable.merge(getLogs(userToken, Logs.FRIENDS), ownLogs) : Observable.empty();
-        final Single<List<LogEntry>> mergedLogs = Single.zip(logs.toList(), specialLogs.toList(),
-                (logEntries, specialLogEntries) -> {
+        final Observable<LogEntry> friendLogs = Settings.isFriendLogsWanted() ?
+                getLogs(userToken, Logs.FRIENDS).cache() : Observable.empty();
+
+        final List<LogEntry> ownLogsFromDb;
+        if (!ownLogs.isEmpty().blockingGet()) {
+            ownLogsFromDb = DataStore.loadLogsOfAuthor(cache.getGeocode(), GCConnector.getInstance().getUserName(), true);
+        } else {
+            ownLogsFromDb = Collections.emptyList();
+        }
+
+        final Single<List<LogEntry>> ownTimeLogs = Single.zip(logs.toList(), ownLogs.toList(),
```

Comment:
Please help here with correct implementation.
Since I am not familiar with the Observable in Java, I didn't manage to change the time of the ownLogEntry, so the following methods are aware of the change.

This implementation with Single.zip and unused first argument works, but is not good...

Reply:
@eddiemuc / @moving-bits 
I searches the last days for a solution to avoid the unnecessary parameter, but didn't find any.
Can one of you please help me?

Reply:
Now I tried with making the log-date editable and using blockingGet(), that works